overrides/app/components/algolia-product-tile/index.jsx

```diff
@@ -26,6 +26,7 @@ import {
     IconButton
 } from '@chakra-ui/react'
 import DynamicImage from '../dynamic-image'
+import AlgoliaProductColors from "../../pages/algolia-product-list/partials/algolia-product-colors";
```

Comment:
I prefer this style but you've previously changed it to follow the Salesforce one.